modules/parachains/src/lib.rs

```diff
@@ -337,112 +333,113 @@ pub mod pallet {
 				parachains.len() as _,
 			);
 
-			pallet_bridge_grandpa::Pallet::<T, T::BridgesGrandpaPalletInstance>::parse_finalized_storage_proof(
+			let mut storage = GrandpaPallet::<T, I>::storage_proof_checker(
```

Comment:
The problem is that with `pallet_bridge_grandpa::Pallet...`, `cargo fmt` doesn't work because the line is too big no matter what. That's why I added the shortcut.